CMakeLists.txt

```diff
@@ -1,11 +1,7 @@
 # Note: CMake support is community-based. The maintainers do not use CMake
 # internally.
 
-cmake_minimum_required(VERSION 3.5)
-
-if (POLICY CMP0048)
-  cmake_policy(SET CMP0048 NEW)
-endif (POLICY CMP0048)
```

Comment:
Manually setting this policy is only necessary if the minimum is 2.X

It's safe to remove this code since the min is 3.6

https://cmake.org/cmake/help/latest/policy/CMP0048.html#policy:CMP0048